plugins/autolink/plugin.js

```diff
@@ -127,6 +121,10 @@
 				return query.match( editor.config.autolink_urlRegex ) ||
 					query.match( editor.config.autolink_emailRegex );
 			}
+		},
+
+		isSupportedEnvironment: function() {
+			return !CKEDITOR.env.ie || CKEDITOR.env.edge;
```

Comment:
Since Edge has a new chromium face, it should support the plugin now.
BTW, in Edge Chromium `CKEDITOR.env.edge === false` 🙈 . That means it is meaningful in the old Edge only and while we could leave this condition to disable the plugin there, it may be misleading. So since we don't really bother much about edgehtml version of Edge (it's less important for us than e.g. IE11), we can remove this part of conditional statement for the sake of clarity.

Reply:
@Dumluregn please, take a look at this statement. It tells that the feature is supported on all modern browsers and Edge HTML. We shouldn't break the feature on the browser where it works correctly, even if it's the older version.

Reply:
Ok, so this comment can be ignored 👍